models/Staticroute.php

```diff
@@ -215,9 +215,14 @@ public static function create()
      */
     private function getDefaultsArray()
     {
-        $defaults = [];
+        $array = $this->getDefaults();
+        if ($array === null) {
```

Comment:
```suggestion
        if (empty($array)) {
```
Using `empty()` would cover empty strings as well.

Reply:
I don't really use `empty` because it's too lenient (exactly how you show), one exception is checking for property initializations. Accepted your suggestion. :+1: